Source/UserManagement/Domain/DataCollector/DataVerifier/ChangeDataVerifierValidator.cs

```diff
@@ -0,0 +1,20 @@
+using Concepts.DataCollector;
+using Concepts.DataVerifier;
+using Dolittle.Commands.Validation;
+using FluentValidation;
+
+namespace Domain.DataCollector.DataVerifier
+{
+    public class ChangeDataVerifierValidator : CommandInputValidatorFor<ChangeDataVerifier>
+    {
+        public ChangeDataVerifierValidator()
+        {
+            RuleFor(_ => _.DataCollectorId)
+                .NotEmpty().WithMessage("Data Collector Id must be set")
+                .SetValidator(new DataCollectorIdValidator());
+            //RuleFor(_ => _.DataVerifierId)
```

Comment:
Outcommented as the serialization of the DataVerifierId (concept as) didn't work. Kept the code here as we should use a DataVerifierId and not a Guid when it works

Reply:
Could you add an Issue describing the issue [here](https://github.com/dolittle-extensions/ReadModels.MongoDB/issues) with a reference to this file and how we can reproduce it ??

Reply:
@woksin can you make the issue? You know more about the details than I.